configs/example/apu_se.py

```diff
@@ -1008,6 +1010,45 @@ def find_file(base_list, rel_path):
 
 exit_event = m5.simulate(maxtick)
 
+while True:
+    if (
+        exit_event.getCause() == "m5_exit instruction encountered"
+        or exit_event.getCause() == "user interrupt received"
+        or exit_event.getCause() == "simulate() limit reached"
+        or "exiting with last active thread context" in exit_event.getCause()
+    ):
+        print(f"breaking loop due to: {exit_event.getCause()}.")
+        break
+    elif "checkpoint" in exit_event.getCause():
+        assert args.checkpoint_dir is not None
+        m5.checkpoint(args.checkpoint_dir)
+        print("breaking loop with checkpoint")
+        break
+    elif "GPU Kernel Completed" in exit_event.getCause():
+        print("GPU Kernel Completed dump and reset")
+        m5.stats.dump()
+        m5.stats.reset()
+    elif "GPU Blit Kernel Completed" in exit_event.getCause():
+        print("GPU Blit Kernel Completed dump and reset")
+        m5.stats.dump()
+        m5.stats.reset()
+    elif "Skipping GPU Kernel" in exit_event.getCause():
+        print("Skipping GPU Kernel dump and reset")
+        m5.stats.dump()
+        m5.stats.reset()
```

Comment:
You should never see this event since the kernel to skip to is not set in the GPUCommandProcessor above.  I would probably also not dump stats here and just continue as no kernel was simulated, however I don't know the exact use case you are going for.

Reply:
Yeah, @yuxiaojia this should probably be removed then -- @abmerop did the kernel skip only get added for GPUFS?

Reply:
Yes, it looks like it was only added for GPUFS